images/tempest-container/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos/centos:stream9
+USER root
+RUN dnf install -y gcc git jq make python3 python3-pip python-setuptools python-requests
+RUN git clone https://git.openstack.org/openstack/tripleo-repos && \
+    pushd tripleo-repos && \
+    python3 setup.py install && \
+    popd && \
+    /usr/local/bin/tripleo-repos current-tripleo-dev
+
+RUN dnf update -y && \
+    dnf install -y python3-tempestconf openstack-tempest openstack-tempest-all && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf/*
+
+RUN git clone https://git.openstack.org/openstack/openstack-tempest-skiplist && \
+    pushd openstack-tempest-skiplist && \
+    python3 -m pip install . && \
+    popd
+RUN curl -s -L https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz | tar xvzf - -C /usr/local/bin oc
```

Comment:
yes, this can happen if new versions of oc client deprecate something. But I don't think that would happen within the same major release (v4).
You mentioned that in base images we don't install oc. The reason is to keep the base root image smaller, since we reuse base root in pre-commit for instance.
But yes, it is possible to install oc client later, in ci-operator config. Do you think that is better to move oc install to there?

Reply:
>> The reason is to keep the base root image smaller, since we reuse base root in pre-commit for instance.

ack thanks, good to know we have a reason

>> But yes, it is possible to install oc client later, in ci-operator config. Do you think that is better to move oc install to there?

As we will not use this tempest image for other jobs like pre-commit, I think we can keep it here. This pattern is more readable.